pkg/bundle/bundle_test.go

```diff
@@ -47,15 +47,15 @@ import (
 	"github.com/cert-manager/trust-manager/test/gen"
 )
 
-func testEncodeJKS(t *testing.T, data string) []byte {
+func testEncodePKCS12(t *testing.T, data string) []byte {
 	t.Helper()
 
 	certPool := util.NewCertPool()
 	if err := certPool.AddCertsFromPEM([]byte(data)); err != nil {
 		t.Fatal(err)
 	}
 
-	encoded, err := truststore.NewJKSEncoder(trustapi.DefaultJKSPassword).Encode(certPool)
+	encoded, err := truststore.NewPKCS12Encoder(trustapi.DefaultPKCS12Password, trustapi.LegacyRC2PKCS12Profile).Encode(certPool)
```

Comment:
suggestion: as discussed in standup 2025-07-18, if we can change this to the Modern profile by default I'm happy to merge this!

Reply:
Fixed, PTAL!